numba_cuda/numba/cuda/cuda_paths.py

```diff
@@ -78,26 +219,46 @@ def _cuda_home_static_cudalib_path():
         return ("lib64",)
 
 
+def _get_cudalib_wheel():
+    """Get the cudalib path from the NVCC wheel."""
+    site_paths = [site.getusersitepackages()] + site.getsitepackages()
+    libdir = not IS_WIN32 and "lib" or "bin"
```

Comment:
This one too (`libdir = "bin" if IS_WIN32 else "lib"`)

(What you have right now is a real brain teaser!)